### ISSUE TITLE
Getting empty allure report

### ISSUE THREAD

**Issue by sonalidwivedi** [Tallyb/cucumber-playwright#518]
I am using allure-playwright with typescript framework and have followed same steps for generating allure report. After execution, allure-report folder is getting created but there is nothing inside data -> test-cases and i am getting "ALLURE REPORT UNKNOWN". I am using  below dependencies:
"@cucumber/cucumber": "^8.7.0",  
"playwright": "^1.26.0", 
"allure-cucumberjs": "^2.0.0-beta.19",

Attaching relevant files for the reference. Please let me know if i am missing any detail.
<img width="505" alt="Screenshot 2022-10-19 at 09 36 24" src="https://user-images.githubusercontent.com/114146513/196627008-2e5dc227-6d28-43e7-a06d-28b2b5f055f9.png">
<img width="917" alt="Screenshot 2022-10-19 at 09 36 50" src="https://user-images.githubusercontent.com/114146513/196627012-eddc3869-8a68-417a-829d-7a2933fc2de5.png">
<img width="633" alt="Screenshot 2022-10-19 at 09 36 56" src="https://user-images.githubusercontent.com/114146513/196627014-2acc4589-0115-404a-bace-b23837e7f05c.png">
<img width="772" alt="Screenshot 2022-10-19 at 09 37 48" src="https://user-images.githubusercontent.com/114146513/196627017-bdd37514-4f32-46f7-9705-553521288934.png">
<img width="1836" alt="Screenshot 2022-10-19 at 09 38 01" src="https://user-images.githubusercontent.com/114146513/196627019-20dc4743-3676-4d3b-be5d-f6c0c2632678.png">
<img width="635" alt="Screenshot 2022-10-19 at 09 30 00" src="https://user-images.githubusercontent.com/114146513/196625255-7d612629-7c6b-4bd9-a257-a7c2758d29ad.png">

**Comment by sonalidwivedi** [comment]
I have also cloned the same code but allure-result folder is not getting generated.

**Comment by Tallyb** [comment]
https://github.com/Tallyb/cucumber-playwright#to-choose-a-reporter

**Comment by sonalidwivedi** [comment]
Hi Tallyb,
Thank you for your reply! I have already used USE_ALLURE=1 in test.yml file as an env parameter and then using "npx cucumber-js && npm run allure" to execute the test but getting report with 0 test cases. It says Allure Report Unknown.

**Comment by Tallyb** [comment]
did you make sure allure is installed? does it work locally?

**Comment by sonalidwivedi** [comment]
Yes, "allure-commandline": "^2.18.1","allure-cucumberjs": "2.0.0-beta.19", is installed. Locally it does not work, i am getting empty report only

**Comment by sonalidwivedi** [comment]
Hi @Tallyb ,
Please help me with this issue.

**Comment by Tallyb** [comment]
Make a repro that I can test

**Comment by Tallyb** [comment]
The error is quite clear: no directory. Create the reports/allure-results folder

**Comment by sonalidwivedi** [comment]
Hi @Tallyb ,

Already tried creating allure-results folder. Still getting the same error.
<img width="1690" alt="Screenshot 2022-10-23 at 15 32 55" src="https://user-images.githubusercontent.com/114146513/197395250-eeaebd40-4eb4-42d0-85fd-bbf9111c23d1.png">
<img width="374" alt="Screenshot 2022-10-23 at 15 33 17" src="https://user-images.githubusercontent.com/114146513/197395254-5e068a1b-532c-4f97-b111-1deeee28acc7.png">
<img width="785" alt="Screenshot 2022-10-23 at 15 33 31" src="https://user-images.githubusercontent.com/114146513/197395256-88aecff9-8d1d-49bf-b5bd-9892e6d6da0c.png">

**Comment by Tallyb** [comment]
Is the folder in the comment fill? Any chance it is folder permissions?

**Comment by sonalidwivedi** [comment]
You mean Allure-Report folder? It looks like this. I do not see any permission issue
<img width="504" alt="Screenshot 2022-10-23 at 15 47 59" src="https://user-images.githubusercontent.com/114146513/197395922-df8359df-e850-40c2-afe0-03791b67bb6c.png">

**Comment by sonalidwivedi** [comment]
I just checked allure-report in finder. It looks like this.
<img width="1487" alt="Screenshot 2022-10-23 at 16 27 13" src="https://user-images.githubusercontent.com/114146513/197397710-30a34591-8e22-4780-ade0-7f76a86c1b9c.png">

**Comment by sonalidwivedi** [comment]
Hi @Tallyb ,

Anything that i can try for resolving this issue?

**Comment by sonalidwivedi** [comment]
Hi @Tallyb,

Please suggest if you have any resolution, it is bit urgent.

**Comment by javillegas2** [comment]
The problem is solved by installing dotenv and including
import dotenv from "dotenv";
dotenv.config();
because apparently it was not reading the configuration made in the .env

**Comment by NerdishShah** [comment]
> The problem is solved by installing dotenv and including import dotenv from "dotenv"; dotenv.config(); because apparently it was not reading the configuration made in the .env

Those who are using .env file should follow @javillegas2 steps mentioned above and for others pass the CLI option as 

`USE_ALLURE=1 npm test` (Mac & Linux)
`set USE_ALLURE=1&&npm test` (Windows)

or update npm scripts in package.json as
`"test": "USE_ALLURE=1 cucumber-js",` (Mac & Linux)
`"test": "set USE_ALLURE=1&&cucumber-js",` (Windows)
